src/index.tsx

```diff
@@ -218,15 +233,64 @@ class Cropper extends React.Component<CropperProps, State> {
 
   computeSizes = () => {
     const mediaRef = this.imageRef || this.videoRef
+
     if (mediaRef && this.containerRef) {
       this.containerRect = this.containerRef.getBoundingClientRect()
+      const containerAspect = this.containerRect.width / this.containerRect.height
+      const naturalWidth = this.imageRef?.naturalWidth || this.videoRef?.videoWidth || 0
+      const naturalHeight = this.imageRef?.naturalHeight || this.videoRef?.videoHeight || 0
+      const isMediaScaledDown = mediaRef.width < naturalWidth || mediaRef.height < naturalHeight
+      const mediaAspect = naturalWidth / naturalHeight
+
+      // We do not rely on the offsetWidth/offsetHeight if the media is scaled down
```

Comment:
I don't know where the issue is but when testing this version with the sandbox showing the "video example", I noticed that the container bounds are not respected anymore when dragging: https://codesandbox.io/s/react-easy-crop-for-videos-forked-w2bl6

![image](https://user-images.githubusercontent.com/2678610/142938483-c3da3e70-ac45-4bcc-8284-c60a04763114.png)


If you don't manage to fix this, I'm totally fine with keeping the original logic if media is a video. 😉

Reply:
Should be fixed now - https://codesandbox.io/s/react-easy-crop-for-videos-forked-sljkb